node/src/components/contract_runtime.rs

```diff
@@ -435,6 +451,69 @@ where
                 }
                 .ignore()
             }
+            Event::Request(ContractRuntimeRequest::ReadTrie {
+                trie_key,
+                responder,
+            }) => {
+                trace!(?trie_key, "read_trie request");
+                let engine_state = Arc::clone(&self.engine_state);
+                let metrics = Arc::clone(&self.metrics);
+                async move {
+                    let correlation_id = CorrelationId::new();
+                    let result = task::spawn_blocking(move || {
+                        let start = Instant::now();
+                        let result = engine_state.read_trie(correlation_id, trie_key);
+                        metrics.read_trie.observe(start.elapsed().as_secs_f64());
+                        result
+                    })
+                    .await
+                    .expect("should run");
+                    trace!(?result, "step response");
+                    responder.respond(result).await
+                }
+                .ignore()
+            }
+            Event::Request(ContractRuntimeRequest::PutTrie { trie, responder }) => {
+                trace!(?trie, "put_trie request");
+                let engine_state = Arc::clone(&self.engine_state);
+                let metrics = Arc::clone(&self.metrics);
+                async move {
+                    let correlation_id = CorrelationId::new();
+                    let result = task::spawn_blocking(move || {
+                        let start = Instant::now();
+                        let result = engine_state.put_trie(correlation_id, trie);
+                        metrics.read_trie.observe(start.elapsed().as_secs_f64());
+                        result
+                    })
+                    .await
+                    .expect("should run");
+                    trace!(?result, "step response");
+                    responder.respond(result).await
+                }
+                .ignore()
+            }
+            Event::Request(ContractRuntimeRequest::MissingTrieKeys {
+                trie_key,
+                responder,
+            }) => {
+                trace!(?trie_key, "missing_trie_keys request");
+                let engine_state = Arc::clone(&self.engine_state);
+                let metrics = Arc::clone(&self.metrics);
+                async move {
+                    let correlation_id = CorrelationId::new();
+                    let result = task::spawn_blocking(move || {
+                        let start = Instant::now();
+                        let result = engine_state.missing_trie_keys(correlation_id, trie_key);
+                        metrics.read_trie.observe(start.elapsed().as_secs_f64());
+                        result
+                    })
+                    .await
+                    .expect("should run");
+                    trace!(?result, "step response");
```

Comment:
The message in this (and the two other) `trace` are probably copied over and need updating.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/752/commits/a2dd1929fee7b4e568836c9c2744785457c8ce86